docker-compose.yml

```diff
@@ -1,6 +1,8 @@
 version: "3"
 services:
   iaso:
+    stdin_open: true
+    tty: true
```

Comment:
What is this for ?

Reply:
This allows to use an interactive shell with Docker, useful to e.g. run a Python debugger.